frb/utils.py

```diff
@@ -34,3 +36,96 @@ def loadyaml(filename):
         data = ayaml.load(infile)
     # Return
     return data
+
+
+def radec_to_coord(radec):
+    """ Converts one of many of Celestial Coordinates
```

Comment:
Isn't this general enough to go in linetools? Isn't already there?

Reply:
I don't want linetools to be a dependency for this Repo